modules/ballerina-test/src/test/java/org/ballerinalang/test/lang/constant/ConstantAccessTest.java

```diff
@@ -16,14 +16,13 @@
  *  under the License.
  */
 
-package org.ballerinalang.model.constant;
+package org.ballerinalang.test.lang.constant;
 
```

Comment:
Instead of test.lang.constant package shall we move this to something like test.types.constants?